### DISCUSSION TITLE
Question about "Removing 1.0 legacy boards from releases and completely removing Heltec wireless capsule"

### DISCUSSION THREAD

**Discussion by gatlinnewhouse** [meshtastic/firmware#5448] [General]
What was the reason for the removal? Also there still seems to be `HELTEC_CAPSULE_SENSOR_V3` conditional compilation flags for the board in the firmware, should those be removed?

**Comment by 0x5c** [comment]
The reasoning is still unexplained, but it's a bit moot now since it was clarified that [support for the Capsule v3 is not being removed](https://github.com/meshtastic/meshtastic/pull/1628#issuecomment-2549139737), and since the compiled firmware [is back in releases](https://github.com/meshtastic/firmware/commit/46eab20a9025d22d97b84eeb9ad96f7e5170e43c) since 2.5.16.

The removal appears to have been a mistake, from what I can see.

**Reply by gatlinnewhouse** [reply]
Thank you for looking into this! I forgot to follow up.
